authorizer-app/src/app/components/rest-source-authorization/rest-source-user-list-delete-dialog.html

```diff
@@ -1,7 +1,8 @@
 <h1 mat-dialog-title>WARNING!</h1>
 <div mat-dialog-content>
   <p>Are you sure you want to delete the user with- <strong>id:</strong> {{data.id}},
-    <strong>user-id:</strong> {{data.userId}} and <strong>project-id:</strong> {{data.projectId}}?
+    <strong>user-id:</strong> {{data.externalId? data.externalId : data.userId}}
```

Comment:
If the external user ID is shown, it would still be helpful to see the actual user ID between brackets, i.e. `blootsvoets (abcde-aefa)`